packages/regl-worldview/src/commands/Arrows.js

```diff
@@ -7,17 +7,50 @@
 //  You may not use this file except in compliance with the License.
 
 import { vec3, quat } from "gl-matrix";
+import fromPairs from "lodash/fromPairs";
+import omit from "lodash/omit";
 import * as React from "react";
 
 import type { Arrow } from "../types";
 import { pointToVec3, vec3ToPoint, orientationToVec4, vec4ToOrientation } from "../utils/commandUtils";
+import { nonInstancedGetChildrenForHitmap } from "../utils/getChildrenForHitmapDefaults";
+import { SUPPORTED_MOUSE_EVENTS, type CommonCommandProps } from "./Command";
 import Cones from "./Cones";
 import Cylinders from "./Cylinders";
 
 const UNIT_X_VECTOR = Object.freeze([0, 0, 1]);
 
-export default class Arrows extends React.PureComponent<{ children: Arrow[] }> {
+type Props = {
+  children: Arrow[],
+  ...CommonCommandProps,
+};
+
+export default class Arrows extends React.PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+
+    // Wrap mouse events so that any cylinder/cone events are propagated correctly.
+    for (const eventName of SUPPORTED_MOUSE_EVENTS) {
```

Comment:
Nice. I imagine we'd use this mapping logic a lot for composite components. Let's extract it to a util function and export it? Also would be nice to convert `Arrows` to a function.

Reply:
Changed this to instead write a new version of `getChildrenForHitmap`